exist-parent/pom.xml

```diff
@@ -92,8 +92,8 @@
     </mailingLists>
 
     <properties>
-        <project.build.source>1.8</project.build.source>
-        <project.build.target>1.8</project.build.target>
+        <project.build.source>17</project.build.source>
```

Comment:
please use the new style: 
<img width="1196" alt="image" src="https://user-images.githubusercontent.com/1700062/192595265-078fba04-62f2-4538-b523-25183e698b93.png">

Reply:
I have kept them this way, as the property you have suggested is directly about the maven-compiler-plugin, whereas our properties are more generic and used for move than just the maven-compiler-plugin.

Reply:
euh? h,,, based om documentation the --release flag on java solves a few 'issues' as well? anyway... makes no difference I assume

Reply:
https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html  "This means that, unlike the [-source and -target options](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html), the compiler will detect and generate an error when using APIs that don't exist in previous releases of Java SE."